modules/nw-metallb-frr-k8s-merge-multiple-configurations.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-frr-k8s.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-metallb-frr-k8s-merge-multiple-configurations_{context}"]
+= How FRR-K8s merges multiple configurations
+
+In a case where multiple users add configurations that select the same node, `FRR-K8s` merges the configurations.
+Each configuration can only extend others. 
+This means that it is possible to add a new neighbor to a router, or to advertise an additional prefix to a neighbor, but not possible to remove a component added by another configuration.
+
+[id="nw-metallb-frr-k8s-merge-multiple-configuration-conflicts_{context}"]
+== Configuration conflicts
+
+Certain configurations can cause conflicts, leading to errors, for example:
+
+* different ASN for the same router (in the same VRF)
+* different ASN for the same neighbor (with the same IP / port)
+* multiple BFD profiles with the same name but different values
+
+When the daemon finds an invalid configuration for a node, it reports the configuration as invalid and reverts to the previous valid `FRR` configuration.
+
+[id="nw-metallb-frr-k8s-merge-multiple-configurations-merging_{context}"]
+== Merging
+
+When merging, it is possible to:
+
+* Extend the set of IPs you want to advertise to a neighbor.
+* Add an extra neighbor with its set of IPs.
+* Extend the set of IPs to which you want to associate a community.
+* Allow incoming routes for a neighbor.
+
+Each configuration must be self contained. This means, for example, that it is not possible to allow prefixes that are not defined in the router section, leveraging prefixes coming from another configuration.
```

Comment:
Is "for example" necessary here? 

I'm confused by the last half of this sentence: leveraging prefixes coming from another configuration. I'm not sure how it relates to the rest of the sentence. Could it be

```suggestion
Each configuration must be self contained. This means, for example, that it is not possible to allow prefixes that are not defined in the router section, or to leverage prefixes coming from another configuration.
```

Reply:
sorry. That wasn't clear I've changed to 
"....it is not possible to allow prefixes that are not defined in the router section _by_ leveraging prefixes coming from another configuration."